src/permutations.jl

```diff
@@ -193,9 +237,9 @@ end
 
 # Parity of permutations
 
-const levicivita_lut = cat([0 0  0;  0 0 1; 0 -1 0],
-                           [0 0 -1;  0 0 0; 1  0 0],
-                           [0 1  0; -1 0 0; 0  0 0]; dims=3)
+const levicivita_lut = cat([0 0 0; 0 0 1; 0 -1 0],
+    [0 0 -1; 0 0 0; 1 0 0],
```

Comment:
the original indentation was better IMO, but I would put a newline after `;` and before `dims=3` for clarity.

Reply:
implemented